tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -602,9 +604,17 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id, HR
                                                                                 &promise);
 
             // wait and get the value from the future<ULONG>
-            const auto& numReJITs = future.get();
+            const auto status = future.wait_for(100ms);
```

Comment:
There are multiple places in `cor_profiler.cpp` where we perform this pattern, can you also update those?

Reply:
Right. In https://github.com/DataDog/dd-trace-dotnet/pull/3650 you updated another future so I included it in https://github.com/DataDog/dd-trace-dotnet/pull/3664/commits/74722b1deb608c56c64267befc10e4627b8db613.
There are other futures but you didn't touch them in your PR, I assume that's because they're not called from ModuleLoadFinished?

Reply:
Right, the other future usages are directly from PInvokes that are initiating by the tracer